plugins/modules/zos_fetch.py

```diff
@@ -91,27 +88,27 @@
     suboptions:
       from:
         description:
-            - The encoding to be converted from
+            - The encoding to be converted from.
         required: true
         type: str
       to:
         description:
-            - The encoding to be converted to
+            - The encoding to be converted to.
         required: true
         type: str
 notes:
     - When fetching PDS(E) and VSAM data sets, temporary storage will be used
       on the remote z/OS system. After the PDS(E) or VSAM data set is
-      successfully transferred, the temporary data set will deleted. The size
+      successfully transferred, the temporary data set will be deleted. The size
       of the temporary storage will correspond to the size of PDS(E) or VSAM
```

Comment:
...to the size of the the PS(sequential data set), PDS, PDSE, member of a PDS/PDSE or
KSDS(VSAM data set).

@RadhaV2020 - I think there is other places PDS(E) is used not showing up in the diff, I think you have a good idea how to work around that....